### PR TITLE
Follower-points-fix

### PR DESCRIPTION
**Pull request by Tsearo**
Fix followed user points when user is appended to end of list to fix issue seen below
image

### CODE REVIEW

File: lib/render/leaderboard.php [hunk 1]
```diff
@@ -226,7 +226,7 @@ function RenderScoreLeaderboardComponent(string $user, bool $friendsOnly, int $n
                     echo GetUserAndTooltipDiv($userData[0]['User']);
                     echo "</td>";
                     if ($j == 0) {
-                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['Points'] . "</a>";
+                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['HardcorePoints'] . "</a>";
                     } else {
                         echo "<td>" . $userData[0]['Points'];
```
Comment (Jamiras) [inline lib/render/leaderboard.php:231]: I'm guessing this needs updated too.

Comment (Tsearo) [reply lib/render/leaderboard.php]: Ah shoot yeah you're right thanks!

File: lib/render/leaderboard.php [hunk 2]
```diff
@@ -226,9 +226,9 @@ function RenderScoreLeaderboardComponent(string $user, bool $friendsOnly, int $n
                     echo GetUserAndTooltipDiv($userData[0]['User']);
                     echo "</td>";
                     if ($j == 0) {
-                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['Points'] . "</a>";
+                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['HardcorePoints'] . "</a>";
                     } else {
-                        echo "<td>" . $userData[0]['Points'];
+                        echo "<td>" . $userData[0]['HArdcorePoints'];
```
Comment (luchaos) [inline lib/render/leaderboard.php:231]: `HardcorePoints`

Comment (Tsearo) [reply lib/render/leaderboard.php]: I do this mistake too often...thanks again

Comment (luchaos) [reply lib/render/leaderboard.php]: No worries! common typo for me as well. just like like "acheivements" :D not sure i can say i like typing that word...

**Review by luchaos — APPROVED** [review]

**Pull request merged**